docs/src/pages/quasar-cli/developing-capacitor-apps/preparation.md

```diff
@@ -59,7 +59,15 @@ In order to develop/build a Mobile app, we need to add the Capacitor mode to our
 $ quasar mode add capacitor
 ```
 
-## 3. Start Developing
+## 3. Install Capacitor
+
+Install Capacitor using npm ([Capacitor installation instructions](https://capacitorjs.com/docs/getting-started)):
+
+```bash
+$ npm install @capacitor/core @capacitor/cli
```

Comment:
Why would someone need install them? They are already in package dependencies: see [here](https://github.com/quasarframework/quasar/blob/dev/app/templates/capacitor/package.json#L8-L9).